rest-api/src/test/java/com/cloudbees/workflow/rest/endpoints/JobAndRunAPITest.java

```diff
@@ -541,8 +541,8 @@ public void test_unstable_basic_flow() throws Exception {
         assertRunPassesSanity(build.get(), run, true);
         Assert.assertEquals(StatusExt.UNSTABLE, run.getStatus());
         Assert.assertEquals(3, run.getStages().size());
-        Assert.assertEquals(StatusExt.UNSTABLE, run.getStages().get(0).getStatus());
-        Assert.assertEquals(StatusExt.UNSTABLE, run.getStages().get(1).getStatus());
+        Assert.assertEquals(StatusExt.SUCCESS, run.getStages().get(0).getStatus());
+        Assert.assertEquals(StatusExt.SUCCESS, run.getStages().get(1).getStatus());
```

Comment:
I am unsure what plugin update caused this change, best guess is `pipeline-graph-analysis` or `workflow-api`. Test probably needs to be reevaluated.